src/target/riscv/riscv.h

```diff
@@ -127,6 +127,8 @@ extern int riscv_reset_timeout_sec;
 extern bool riscv_use_scratch_ram;
 extern uint64_t riscv_scratch_ram_address;
 
+extern bool riscv_prefer_sba;
```

Comment:
so this is global for all targets? Probably fine since this isn't expected to be used much, though it seems like it would make sense as a per-target thing.

Reply:
I don't know how to do per-target settings. There may be an example of it in the code somewhere, but the things I can imagine aren't very pretty.